content/articles/domain-access-control.md

```diff
@@ -29,56 +31,86 @@ With the Domain Access Control feature, you can set the level of access for each
 
 ## Setting access for a team member
 
-To set a team member's role for multiple domains at once, go to the Account > Members page. Click on the Manage Access button to configure the team member's access. You can give the team member [Full Access](#full-access) or [Limited Access](#limited-access).
+To set a team member's role for multiple domains at once:
+
+1. Go to your Account page. 
+1. Click the **Members and seats** tab on the left side. On the Members card, you'll find each team member along with **Manage Access** and **Remove** buttons.
+1. Click **Manage Access** on a specific team member to configure their access level as [**Full Access**](#full-access) or [**Limited Access**](#limited-access).
 
-![Account Member's Page](/files/domain-access-control-members-page.png)
+<!--- needs screenshot -->
```

Comment:
https://drive.google.com/file/d/131crbAnY2TMdDH7k_U8PoOmbTcNclXYs/view?usp=sharing

Reply:
@dperrymorrow the wrong screenshot was still showing up in preview — still need to add this screenshot there (reverted it to the tag for visibility)